o4/o4.py

```diff
@@ -844,8 +844,7 @@ def man(cmd):
         if seed_move:
             syncit += " --move"
 
-    cmd = (f"{fstat} | {o4bin} keep --deletes"
-           f"| {manifold_big} {o4bin} drop --checksum"
+    cmd = (f"{fstat} | {o4bin} drop --not-deletes --existence"
```

Comment:
Keep. Drop. What's the big difference?

Reply:
"keep" determines the final result by the logical "and" of each qualifier supplied, while "drop" uses "or".